docs/getting-started/readme.md

```diff
@@ -4,6 +4,8 @@ items:
   - forking.md
   - config-env.md
   - db.md
+  - start-app.md
+  - workflow.md
```

Comment:
Everything seems cool! I can't find start-app.md and workflow.md in the PR though

Reply:
GOOD CATCH! just added.